integration/src/test/java/org/mule/test/integration/exceptions/ErrorHandlerLifecycleTestCase.java

```diff
@@ -49,6 +51,14 @@ protected String getConfigFile() {
   @Named("flowD")
   private FlowConstruct flowD;
 
+  @Inject
```

Comment:
Add also tests indicating what the correct behavior for the start and initialise phase should be in the case of a GlobalErrorHandler.